resources/js/beatmapsets-show/info.tsx

```diff
@@ -111,127 +111,129 @@ export default class Info extends React.Component<Props> {
-              </h3>
-              <a
-                className='beatmapset-info__link'
-                href={route('beatmapsets.index', { g: this.controller.beatmapset.genre.id })}
-              >
-                {this.controller.beatmapset.genre.name}
-              </a>
-            </div>
-
-            <div className='beatmapset-info__half-entry'>
-              <h3 className='beatmapset-info__header'>
-                {trans('beatmapsets.show.info.language')}
-              </h3>
-              <a
-                className='beatmapset-info__link'
-                href={route('beatmapsets.index', { l: this.controller.beatmapset.language.id })}
-              >
-                {this.controller.beatmapset.language.name}
-              </a>
-            </div>
-          </div>
 
-          {this.controller.tags.userTags.length > 0 &&
-            <div className='beatmapset-info__row beatmapset-info__row--value-overflow'>
-              <h3 className='beatmapset-info__header'>
-                {trans('beatmapsets.show.info.user_tags')}
-              </h3>
-              <div className='beatmapset-info__value-overflow'>
-                {this.controller.tags.userTags.map((tag) => (
-                  <React.Fragment key={tag.name}>
-                    <a
-                      className='beatmapset-info__link'
-                      href={route('beatmapsets.index', { q: makeSearchQueryOption('tag', tag.name) })}
-                    >
-                      {tag.name}
-                    </a>
-                    {' '}
-                  </React.Fragment>
-                ))}
+              <div className='beatmapset-info__half-entry'>
+                <h3 className='beatmapset-info__header'>
+                  {trans('beatmapsets.show.info.language')}
+                </h3>
+                <a
+                  className='beatmapset-info__link'
+                  href={route('beatmapsets.index', { l: this.controller.beatmapset.language.id })}
+                >
+                  {this.controller.beatmapset.language.name}
+                </a>
               </div>
             </div>
-          }
 
-          {this.controller.tags.mapperTags.length > 0 &&
-            <div className='beatmapset-info__row beatmapset-info__row--value-overflow'>
-              <h3 className='beatmapset-info__header'>
-                {trans('beatmapsets.show.info.mapper_tags')}
-              </h3>
-              <div className='beatmapset-info__value-overflow'>
-                {this.controller.tags.mapperTags.map((tag, i) => (
-                  <React.Fragment key={`${tag}-${i}`}>
-                    <a
-                      className='beatmapset-info__link'
-                      href={route('beatmapsets.index', { q: tag })}
-                    >
-                      {tag}
-                    </a>
-                    {' '}
-                  </React.Fragment>
-                ))}
+            {this.controller.tags.userTags.length > 0 &&
```

Comment:
not directly related to this pr but apparently user tags should still be shown for converts